src/main/java/com/dwbtest/App.java

```diff
@@ -0,0 +1,12 @@
+package com.dwbtest;
+
+/**
+ * Hello world!
+ *
+ */
+public class App {
+    public static void main( String[] args )
+    {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) '{' should be on the previous line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.blocks.LeftCurlyCheck)